specification/dsdl/architecture.tex

```diff
@@ -230,6 +228,18 @@ \subsection{Elements of data type definition}
 or assertions (special statements introduced by data type designers
 which are statically validated by DSDL processing tools).
 
+Service type definitions are a special case:
```

Comment:
why not just say that service types are not serializable types? We'll just say they are a namespace with strict rules. We might even allow them to be different .dsdl files in the future; who knows?

Reply:
Moving service types into a different category is a massive can of worms :canned_food: :snake: 

They have attributes that are inherent to serializable types like name, version, fixed port-ID, and deprecation status, so they are closer to being a special category than namespaces or regular serializable types. Do we want it changed in this PR? Should we postpone it until later? Do we keep it as-is? I tend to prefer the latter option right now.

Reply:
Hmph. You are right. Okay, um, just call then service request and/or response types then. If we were to change how we defined service types this would remain the same terminology.